client/client.go

```diff
@@ -369,24 +364,6 @@ func (c *Client) CmdDeposit(status chan *CmdStatus, args ...string) {
 	}
 	// TODO challenge
 	status <- &CmdStatus{Err: errors.New("TODO challenge")}
-	return
-}
-
-func errorReason(ctx context.Context, b *pethchannel.ContractBackend, tx *types.Transaction, blockNum *big.Int, acc accounts.Account) (string, error) {
```

Comment:
do you already use our `ContractBackend.WaitMined` function, which tries to extract revert reasons?

Reply:
This function was unused, therefore i deleted it. The client+operator currently use `abi/bind.WaitMined` but should probably switch to `ContractBackend.ConfirmTransaction`.

Reply:
and we should probably just rename it to `WaitMined` in go-perun :joy:

Reply:
-> #54